lib/pf/dhcp/processor.pm

```diff
@@ -117,12 +120,28 @@ sub _get_redis_client {
         return $self->{redis_client};
     }
     else {
-        my $server = pf::CHI->config->{storage}{redis}{server};
-        $self->{redis_client} = Redis::Fast->new(server => $server, on_connect => \&_on_redis_connect);
+        my $config = _get_redis_config();
+        $self->{redis_client} = pf::Redis->new(%$config, on_connect => \&_on_redis_connect);
         return $self->{redis_client};
     }
 }
 
+
+=head2 _get_redis_config
+
+Get the redis config from pf::CHI
+
+=cut
+
+sub _get_redis_config {
```

Comment:
Not sure what you mean ?
Do you mean where I would see it more ?
I would say in pf::CHI since thats where we get the config from but any generic place is good